paper/tex/data.tex

```diff
@@ -1,39 +1,80 @@
+
+\subsubsection{Training and test set division}
 \label{sec:buzztraining}
-The total initial catalogue covered $400$ square degrees and contained $238$ million galaxies to an apparent magnitude limit of $r\!=\!29$ and spanning the redshift range $0\!<\!z\!\leq\!8.7$.  In order for statistical errors not to dominate, we need less than one million galaxies in our sample.  Several studies claim that only a few tens of thousands of spectra are necessary to calibrate photo-z surveys to Stage IV requirements (e.~g.~\citet{Bernstein:10}, \citet{Masters:2017}).  Therefore, we aim for a final number of training galaxies between $3\times 10^{4}$ and $5\times 10^{4}$ in our sample.
-%This catalogue contained two orders of magnitude more galaxies than were necessary to determine statistics for this study,
-In order to reduce our sample to a reasonable size, we limit our dataset to a subset of $\sim\!16.8$ square degrees selected from five separate spatial regions of the simulation. Systematic problems with galaxy colors above $z\!>\!2$ were observed, so the catalogue was limited to include only galaxies in the redshift range $0\!<\!z\!\leq\!2.0$. A random subset of the the remaining galaxies was chosen, and placed at random into either a ``training'' set ($10$ per cent of the sample), for which the galaxies true redshifts will be supplied, or a ``test'' set (the remaining $90$ per cent of the sample), for which each code will need to predict a redshift PDF for each galaxy.  
+
+Following the findings of \citet{Bernstein:10}, \citet{Masters:2017} that only $~10^{4}$ spectra are necessary to calibrate \pz s to Stage IV requirements, we aimed to set aside a randomly selected training set of $3-5\times 10^{4}$ galaxies, $\sim 10\%$ of the full sample.
+After all cuts described above, we designated the \textit{DC1 training set} of $44\,404$ galaxies for which observed photometry, true SEDs, and true redshifts would be provided to all codes codes and the blinded \textit{DC1 test set} of $399\,356$ galaxies for which photometry alone would be provided to all codes and \pzpdf s would be requested.
 %The resulting catalogues contain $111\,171$ training galaxies and $1\,000\,883$ test galaxies.
-Finally, we restrict our analysis to a sample with an apparent i-band magnitude limit $i<25.3$, which give a signal-to-noise $\sim\,30$ for most galaxies, a cut often referred to as the expected ``LSST Gold Sample''.  This magnitude cut results in a training set with $44\,404$ galaxies and a test set containing $399\,356$ galaxies.  All subsequent results will evaluate this ``gold sample'' test set.  In order to blind results, initially redshifts were not revealed for the ``test'' set, and were only supplied for the training sample galaxies.  This prevented code runners from tweaking results and fitting to the specific test set.
+The \lsst\ photometric filter transmission curves were also considered public information that could be used by any code.
```

Comment:
maybe "the exact form of the photometric filter transmission curves assumed when generating the mock data"